CONTRIBUTING.md

```diff
@@ -26,7 +26,65 @@ guidance below.
 
 Generally speaking, you should fork this repository, make changes in your
 own fork, and then submit a pull-request. All new code should have associated unit
-tests that validate implemented features and the presence or lack of defects. 
-Additionally, the code should follow any stylistic and architectural guidelines 
+tests that validate implemented features and the presence or lack of defects.
+Additionally, the code should follow any stylistic and architectural guidelines
 prescribed by the project. In the absence of such guidelines, mimic the styles
 and patterns in the existing code-base.
+
+## Browser support
+
+We configure [Autoprefixer](#autoprefixer) to support the following list of browsers.
+
+- Latest 2 releases of all browsers including:
+    - Chrome
+    - Firefox
+    - Safari
+    - Internet Explorer
+    - Edge
+    - Opera
+    - iOS Safari
+    - Opera Mini
+    - Android Browser
+    - BlackBerry Browser
+    - Opera Mobile
+    - Chrome for Android
+    - Firefox for Android
+    - Samsung Internet
+- Internet Explorer 9
+- Internet Explorer 8
```

Comment:
Project has reference to Internet Explorer 7 https://github.com/cfpb/retirement/blob/master/gulp/tasks/styles.js#L40

Reply:
@anselmbradford thanks for catching this, updated